gen/suites/reward/main.go

```diff
@@ -24,4 +27,120 @@ func main() {
 			TipsetFunc: minersAwardedNoPremiums,
 		},
 	)
+
+	g.Group("penalties",
+		&VectorDef{
+			Metadata: &Metadata{
+				ID:      "penalize-inexistent-sender-multiple-miners",
+				Version: "v1",
+				Desc:    "verifies that a miner including a message from an inexistent sender is penalized; only the first miner is penalized",
+				Comment: "https://github.com/filecoin-project/lotus/issues/3491",
+			},
+			TipsetFunc: minerPenalized(3, func(v *TipsetVectorBuilder) {
```

Comment:
I'm not sure I understand why we always use 3 miners, but only ever use 1 of them?

Reply:
Creating a miner ends up with three actors: 
- the miner actor.
- the owner actor (account actor).
- the worker actor (account actor).

To do transfers, we use the owner actor of a miner as a shorthand for not having to create more accounts in the system. I think this is what you're referring to.

But the `minerPenalized` skeleton actually simulates that all miners are creating a block with wincount=1, and including all messages.

We then assert that all miner balances have been incremented by the block reward or decremented by the penalties.

Reply:
Ok so just to check we _could_ use `minerCnt=1` in these tests but we're using `minerCnt=3` because we want to ensure correctness when multiple miners are involved?

Reply:
Yeah, exactly. And we did find some unexpected behaviour, which turned out to be OK:  which folks seem to be fine with: https://github.com/filecoin-project/lotus/issues/3491